api/server/server.go

```diff
@@ -554,7 +554,7 @@ func (s *Server) getContainersStats(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	return s.daemon.ContainerStats(vars["name"], boolValue(r, "stream"), ioutils.NewWriteFlusher(w))
+	return s.daemon.ContainerStats(vars["name"], boolValueOrDefault(r, "stream", true), ioutils.NewWriteFlusher(w))
```

Comment:
Technically not; but do they all explicitly use a /v1.xx ?

All the same, if (e.g.) docker-compose wants to support multiple versions of docker, it can become a pita to support, or by default use an older API version.

Not 100% against, just think it should be properly thought through.